packages/eslint-config-custom/index.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
         'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
         'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
         'no-unsafe-optional-chaining': 1,
-        'no-undef': 1,
+        'no-undef': 0,
```

Comment:
Additional explanation, please!

Reply:
There was an issue when use the type that exists in dom.iterable.
In this [raddit site](https://www.reddit.com/r/vuejs/comments/x9utfh/eslint_error_with_global_type_definitions/), there was the pretty same issue, and the solution was removing this rule since this is eslint rule and many cases, typescript will filter this problem.